knowledge-base/generate-table-of-contents-radwordsprocessing.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Generating a Table of Contents in a Merged Document Using RadWordsProcessing
+description: Learn how to merge two Word documents and generate a unified table of contents using the RadWordsProcessing library.
+type: how-to
+page_title: How to Create a Table of Contents in Merged Word Documents with RadWordsProcessing
+slug: generate-table-of-contents-radwordsprocessing
+tags: wordsprocessing, document, processing, merge, table, contents, toc, docx, pdf
+res_type: kb
+ticketid: 1657796
+---
+
+## Environment
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 2024.2.426| RadWordsProcessing |[Desislava Yordanova](https://www.telerik.com/blogs/author/desislava-yordanova)| 
+
+## Description
+
+This article shows how to merge two DOCX documents into a single document and generate a unified TOC that correctly reflects the merged content.  
+
+## Solution
+
+To merge two Word documents and generate a unified TOC, follow these steps:
+
+1. Import the documents using the [DocxFormatProvider]({%slug radwordsprocessing-formats-and-conversion-docx-docxformatprovider%}).
+2. [Merge]({%slug radwordsprocessing-editing-clone-and-merge%}) the documents using the `Merge` method with appropriate `MergeOptions`.
```

Comment:
add link to merge/merg options - https://docs.telerik.com/devtools/document-processing/libraries/radwordsprocessing/editing/clone-and-merge#merging-documents